### PR TITLE
MNT: Re-rendered with conda-smithy 1.3.2.

### PR DESCRIPTION
**Pull request by ngoldbaum**

### CODE REVIEW

**Comment by ngoldbaum** [comment]
@jakirkham any idea what's up with the build failure?

```
unknown key conda-build
Error: 'numpy x.x' requires external setting

./ci_support/run_docker_build.sh returned exit code 1
```

File: ci_support/run_docker_build.sh [hunk 1]
```diff
@@ -41,46 +41,7 @@ conda clean --lock
...
-    set -x
-    export CONDA_NPY=110
-    export CONDA_PY=35
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
-
-    set -x
-    export CONDA_NPY=111
-    export CONDA_PY=35
```
Comment (jakirkham) [inline ci_support/run_docker_build.sh]: Seems the Python and NumPy settings got dropped. That's the problem. Could we please try to do a new re-rendering?

Comment (ngoldbaum) [inline ci_support/run_docker_build.sh]: Hmm, i'm not sure how to generate these lines, they're not produced when I do `conda smithy rerender`. I do see messages about missing circle, appveyor, and anaconda tokens, is that related?

**Comment by jakirkham** [comment]
Canceled all CIs while we sort this out as the CIs are already pretty backlogged ATM.

File: ci_support/run_docker_build.sh (hunk 1 above)
Comment (ngoldbaum) [inline ci_support/run_docker_build.sh]: https://gist.github.com/ngoldbaum/21a65af75bec630764d65c9cc808e79f

Comment (jakirkham) [inline ci_support/run_docker_build.sh]: So we have seen this before<sup>1</sup>. The fix was to get everything from the conda-forge channel in `root`. Maybe try that either in your current Miniconda install or a new one and see if that doesn't work.


1 - https://github.com/conda-forge/conda-smithy/issues/223

Comment (jakirkham) [inline ci_support/run_docker_build.sh]: Maybe try commenting `pyembree` from build while re-rendering. Just make sure to uncomment it once you are done/before committing.

**Comment by jakirkham** [comment]
Before this gets merged, I would try to fix `pyembree`'s compatibility ( https://github.com/conda-forge/pyembree-feedstock/issues/1 ) first.

File: ci_support/run_docker_build.sh (hunk 1 above)
Comment (jakirkham) [inline ci_support/run_docker_build.sh]: Would be great to figure out why that happened. Though it is not crucial ATM. Maybe add an issue on this feedstock about this re-rendering problem and xref it back here. That way we can take a closer look when we have more time.

Comment (jakirkham) [inline ci_support/run_docker_build.sh]: xref: https://github.com/conda-forge/yt-feedstock/issues/9

**Comment by ngoldbaum** [comment]
I'm merging this now to restore compatibility with older OSX versions. We can come back to the pyembree stuff later.
